### PR TITLE
chrono-tz feature: Implement GraphQLType for chrono-tz

### PR DESCRIPTION
**Pull request by nirmangupta**

### CODE REVIEW

**Comment by nirmangupta** [comment]
aims to fix https://github.com/graphql-rust/juniper/issues/516

**Comment by nirmangupta** [comment]
`Test book code examples via skeptics` fails. The errors in the console seem unrelated though?

**Comment by nirmangupta** [comment]
Looking at current open PRs after the new CI and hooks were added, this particular step seems to be failing for all

File: juniper/src/integrations/chrono_tz.rs [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/*!
+
```
Comment (LegNeato) [inline juniper/src/integrations/chrono_tz.rs]: I don't think we need this table. We only had it for chrono due to serde ambiguity .

File: juniper/Cargo.toml [hunk 2]
```diff
@@ -28,6 +28,7 @@ async = ["juniper_codegen/async", "futures"]
 expose-test-schema = []
 default = [
     "chrono",
+    "chrono-tz",
```
Comment (LegNeato) [inline juniper/Cargo.toml]: I'm not sure this meets the level for on by default. Need to think about this some more.

File: juniper/src/integrations/chrono_tz.rs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+*/
+use chrono_tz::Tz;
+
+use crate::{
+    parser::{ParseError, ScalarToken, Token},
+    value::ParseScalarResult,
+    Value,
+};
+
+graphql_scalar!(Tz as "Tz" where Scalar = <S>{
```
Comment (LegNeato) [inline juniper/src/integrations/chrono_tz.rs]: Nit: Add a space after `<S>`

File: juniper/src/integrations/chrono_tz.rs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+    #[test]
+    fn tz_from_input_value_etc_minus() {
+        tz_input_test("Etc/GMT-3", Some(chrono_tz::Etc::GMTMinus3));
+    }
+
+    #[test]
+    fn tz_from_input_value_invalid() {
+        tz_input_test("Abc/Xyz", None);
+    }
```
Comment (LegNeato) [inline juniper/src/integrations/chrono_tz.rs]: Can you add a test for a number, and a string that isn't in the proper "XX/YY" format?

File: juniper/src/integrations/mod.rs [hunk 5]
```diff
@@ -6,6 +6,10 @@ pub mod serde;
 /// GraphQL support for [chrono](https://github.com/chronotope/chrono) types.
 pub mod chrono;
 
+#[cfg(feature = "chrono-tz")]
+/// GraphQL support for [chrono-tz](https://https://github.com/chronotope/chrono-tz) types.
```
Comment (LegNeato) [inline juniper/src/integrations/mod.rs]: There is an extra `https://` here.

File: juniper/Cargo.toml (hunk 2 above)
Comment (nirman28) [reply juniper/Cargo.toml]: fair point. I have removed it from the default feature list

File: juniper/src/integrations/chrono_tz.rs (hunk 3 above)
Comment (nirman28) [reply juniper/src/integrations/chrono_tz.rs]: thanks for noticing this. Wonder why `rustfmt` didn't take care of this 🤔

**Comment by nirman28** [comment]
:( broke all the tests, fixing this

**Comment by nirmangupta** [comment]
@LegNeato while all the tests are passing now, the `chrono-tz` feature is neither part of documentation nor part of CI pipeline. How can I remedy this?

**Comment by nirmangupta** [comment]
@LegNeato while all the tests are passing now, the `chrono-tz` feature is neither part of documentation nor part of CI pipeline. How can I remedy this?

**Comment by nirmangupta** [comment]
@LegNeato could you review this again?

**Review by tyranron — APPROVED** [review]
@nirmangupta sorry for the long delay and thanks once again! 🍻

**Pull request merged**